xamarin/Xamarin.Forms
# [iOS] Fix crash with TimePicker on iOS14

### Description of Change ###

Fix the TimePicker crash on iOS14 by forcing the PreferredDatePickerStyle to UIDatePickerStyle.Wheels.

### Issues Resolved ### 

- fixes #11963

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Use Xcode12, iOS14 device, and Xamarin.iOS for IOS14.
Go to issue 11943, click on the date , make sure the time picker wheels appear and you able to select a time.

make sure it also works on iOS13

### PR Checklist ###
